share/spice/yacht_specs/content.handlebars

```diff
@@ -0,0 +1,32 @@
+<!-- Div for the yacht specifications -->
+<div class='thumb'>
+    
+    <div class='image'>
```

Comment:
Once we add the code for the normalize parameter as you've given it here, the instant answer doesn't seem to appear. Do you have an idea why ?